README.md

```diff
@@ -250,6 +250,36 @@ There are several options for adopting the latest VVV configuration.
       you have something set up that you cannot part with.
 
 ## Miscellaneous Issues
+
+### 04/26/2022
+
+Error: Provisioning fails with the message `fatal: unsafe repository ('/srv/provision/utilities/pmc' is owned by someone else)`. It is not resolved by re-adding the SSH key nor by running the suggested command `git config --global --add safe.directory`.
+
+Fix: Find provision.sh in your VVV directory, e.g. VVV/provision/provision.sh. Before the conditional around like 36, add the following:
+
+```
+git config --global --add safe.directory /srv/provision/utilities/pmc
+git config --global --add safe.directory /srv/provision/utilities/core
+git config --global --add safe.directory /srv/provision/extensions/pmc
+git config --global --add safe.directory /srv/provision/extensions/core
+git config --global --add safe.directory /srv/www/wordpress-trunk
+```
+
+Then, for each site enabled in config.yml, add the following:
+
+```
+git config --global --add safe.directory /srv/www/deadline-com
+git config --global --add safe.directory /srv/www/indiewire-com
+git config --global --add safe.directory /srv/www/robbreport-com
+git config --global --add safe.directory /srv/www/rollingstone-2022-com
+git config --global --add safe.directory /srv/www/sportico-com
+git config --global --add safe.directory /srv/www/rollingstone-com
+git config --global --add safe.directory /srv/www/variety-com
+git config --global --add safe.directory /srv/www/vibe-com
```

Comment:
These can be done with a single command. Note, any of these --add safe.directory commands will only work in `Git v2.35.3+`

```suggestion
git config --global --add safe.directory '*'
```

Reply:
James, this command can be run before the provision command, right? or is it required to be added to the provision.sh file?

Reply:
I believe it is needed in provision.sh because that script is running within the VM (i.e. this command is not needed on your host machine, only in the VM)